docs/data/tree-view/accessibility/accessibility.md

```diff
@@ -31,8 +31,10 @@ The [WAI-ARIA Authoring Practices](https://www.w3.org/WAI/ARIA/apg/patterns/tree
 :::info
 The following key assignments apply to Windows and Linux users.
 
-On macOS replace <kbd class="key">Ctrl</kbd> with <kbd class="key">⌘ Command</kbd>.
-
+On macOS replace:
```

Comment:
I'm not sure how correct we are here. 🤔 
Those keys definitely exist on MacOS and their keyboards:
https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/MQ052?wid=2000&hei=2000&fmt=jpeg&qlt=95&.v=1495129815011
It's only MacBooks with their limited amount of keys or any other keyboard with less keys, which remove less often used keys. 🤔 
So, technically, with this we are educating users about MacOS shortcuts for keys that they MIGHT not have on their keyboard.
I'm not saying that we can't do it. Just questioning if maybe we could put a bit less emphasis on it and instead only link to a MacOS keyboard shortcut page? 🤔

Reply:
It makes sense, we can definitely put those additions as improvements that would have less emphasis than actually different buttons. 👍